OsmAnd/src/net/osmand/plus/measurementtool/MeasurementToolFragment.java

```diff
@@ -2173,6 +2173,12 @@ private void startTrackNavigation() {
 			if (editingCtx.hasRoute() || editingCtx.hasChanges()) {
 				String trackName = getSuggestedFileName();
 				GPXFile gpx = editingCtx.exportGpx(trackName);
+				// come back wpt point to gpx after approximation
```

Comment:
Probably it's not the only place or not correct cause Attach to Roads could be called from Plan Route (MeasureFramgent) & Navigation.